content/brazilian-portuguese/linux-basics/answer-key.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Resoluções - Curso Básico de Linux"
+description: "Resoluções de todos os exercícios"
+date: 2022-09-22
+difficulty: "Intermediário"
+weight: 15
+---
+
+### Atividade 1
+
+```
+whoami
+```
+
+### Atividade 2
+
+```
+pwd
+cd Desktop
+cd..
+```
+
+### Atividade 3
+
+```
+ls -l
+mv Documents/ Downloads/
+ls -l Downloads
+mv Downloads/Documents/ .
+```
+
+### Atividade 4
+
+```
+mkdir files
+touch file1
+vim file1
+mv file1 files
+```
+
+### Atividade 5
+
+```
+chmod u=rwx g=rx o=x file1
+chmod 751 [insert filename]
+```
+
+### Atividade 6
+
+```
+cd
+grep -R "Do. Or do not. There is no try."
+find . -name *.sh
+```
+
+### Atividade 7
+
+```
+head -n 5 malware.sh
+tail -n 5 malware.sh
+cat malware.sh
+more malware.sh
+less malware.sh
+```
+
+### Atividade 8
+
+```
+find . -name *.txt
+grep -R "Ser. Ou não ser. Eis a questão."
```

Comment:
Here also, I guess it's better to keep the original text to avoid wrong results.
```suggestion
grep -R "Do. Or do not. There is no try."
```